code-challenges.test.js

```diff
@@ -17,43 +17,79 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe ( 'When "divisibleByThree" is called', () => {
+    it ( "return '15' is divisible by three", () => {
+        const numberOne = 15
+        const thisWorked = divisibleByThree(numberOne);
+        expect (thisWorked).toEqual(`${numberOne} is divisible by three`)
+    })
+    it ( "return '0' is divisible by three", () => {
+        const numberTwo = 0
+        const thisWorkedAgain = divisibleByThree(numberTwo);
+        expect (thisWorkedAgain).toEqual(`${numberTwo} is divisible by three`)
+    })
+    it ( "return '-7' is not divisible by three", () => {
+        const numberThree = -7
+        const wowThirdTime = divisibleByThree(numberThree);
+        expect (wowThirdTime).toEqual(`${numberThree} is not divisible by three`)
```

Comment:
```suggestion
        expect(wowThirdTime).toEqual(`${numberThree} is not divisible by three`)
```
spaces are not needed after function names for a function call. Not a big deal but it's best practice to be consistent about spacing